corehq/apps/reports/migrations/0008_tableauvisualization_title.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.2.25 on 2022-06-16 18:48
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('reports', '0007_tableauvisualization_upstream_id'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='tableauvisualization',
+            name='title',
+            field=models.CharField(default='', max_length=32, null=True),
```

Comment:
This doesn't match the model anymore, you can remove the `default` argument.

Reply:
(blocking, because I think django will continue to complain that models and the db are out of sync)